Assets/Scripts/GameSelection.cs

```diff
@@ -81,30 +79,91 @@ public GameSelection()
         {
             startColor = Color.gray;
         }
-        tb = new TextButton(new Vector2((UIScaler.GetWidthUnits() - 30) / 2, 19), new Vector2(30, 4f), "Mansions of Madness Second Edition", delegate { MoM(); }, startColor);
+        tb = new TextButton(new Vector2((UIScaler.GetWidthUnits() - 32) / 2, 19), new Vector2(30, 4f), "Mansions of Madness Second Edition", delegate { MoM(); }, startColor);
         tb.button.GetComponent<UnityEngine.UI.Text>().fontSize = UIScaler.GetMediumFont();
         tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0, 0.03f, 0f);
 
         // Draw MoM import button
         if (fcMoM.importAvailable)
         {
-            if (fcMoM.NeedImport())
+            string text = fcMoM.NeedImport() ? "Import Content" : "Reimport Content";
+            tb = new TextButton(new Vector2((UIScaler.GetWidthUnits() - 12) / 2, 23.2f), new Vector2(10, 2f), text, delegate { Import("MoM"); });
+            tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0, 0.03f, 0f);
+        }
+        else // Import unavailable
+        {
+            db = new DialogBox(new Vector2((UIScaler.GetWidthUnits() - 26) / 2, 23.2f), new Vector2(24, 1f), "Unable to locate Mansions of Madness, install via Steam", Color.red);
+            db.AddBorder();
+        }
+
+        // If localization data were imported.
+        if (!fcMoM.NeedImport())
+        {
+            if (game.config.data.Get("MoMConfig") == null)
             {
-                tb = new TextButton(new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 23.2f), new Vector2(10, 2f), "Import Content", delegate { Import("MoM"); });
-                tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0, 0.03f, 0f);
+                // TODO: FFGlookup to get the first line (languages list)
+                string langsString = StringI18n.FFG_LANGS;
+
+                // List of languages
+                game.config.data.Add("MoMConfig","Langs", langsString);
+                // English is the default current language
+                game.config.data.Add("MoMConfig", "currentLang","1");
+                game.config.Save();
             }
-            else // Import OK, can redo
+
+            string[] langs = game.config.data.Get("MoMConfig", "Langs").Split(',');
+            int currentLang = int.Parse(game.config.data.Get("MoMConfig", "currentLang"));
+
+            game.currentLang = currentLang;
+
+            float verticalStart = 12 - ((langs.Length - 1) * 1.25f);
```

Comment:
any reason this isn't:

float verticalStart = 15 - (langs.Length * 1.25f);

Or

float verticalStart = UIScaler.GetVCenter(langs.Length * -1.25f);


For vertical center?